segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/handler/AbstractConnectionListener.java

```diff
@@ -72,14 +82,45 @@
     @Getter
     private final boolean enableTlsReload; // whether to reload TLS certificate when the certificate changes
 
+    private final String type;
```

Comment:
Is this necessary? I think that you can use `this.getClass().getSimpleName()` as input for `healthServiceManager.register` and you can get rid of this field.